main.go

```diff
@@ -79,6 +79,7 @@ func main() {
 				fmt.Printf("(%d bytes)\n", len(contents))
 			}
 
+			path = filepath.ToSlash(path)
```

Comment:
@ncruces Is the `ToSlash` needed here to keep the paths within `files` using `/` separators rather than using `\` on windows?

Reply:
Yes. Which I think makes sense?

Additionally, if you don't do this, you produce a file that doesn't compile (formatting fails), because the template is full of "invalid escape sequences". If you wanted to keep the backslashes, you'd need to escape them, or use a raw string literal (which might not be such a bad idea?)

Reply:
Yup I think this makes sense.